### PR TITLE
UI/UX and some bug fixes

### PR DESCRIPTION
**Pull request by RashaHussein**
*Center Articles and Publishers. 
*Add Write New Article button in profile and categories views
*Clearer edit profile button
*Remove Publishers Bio from Writers Section
*Add navigation buttons to categories and topics views
*Fix save publishers articles count

### CODE REVIEW

File: backend/app/models/article.rb [hunk 1]
```diff
@@ -60,6 +60,7 @@ def word_count
 
   def update_published_articles_count
     self.user.published_articles_count = self.user.published_articles.count
+    self.user.save
```
Comment (mkhatib) [inline backend/app/models/article.rb]: Nice :cat:ch

File: web-client/app/views/main.html [hunk 2]
```diff
@@ -59,15 +59,14 @@
...
           <a ng-href="/profiles/{{user.id}}">
             <h1>{{user.name}}</h1>
-            <h2>{{user.bio}}</h2>
           </a>
           <div class="stats-box">
             <span ng-show="user.published_articles_count">
               <i class="fa fa-pencil-square-o"></i> {{user.published_articles_count}}
             </span>
-            <span ng-show="user.recommendations_count">
+            <!-- <span ng-show="user.recommendations_count">
```
Comment (mkhatib) [inline web-client/app/views/main.html]: Do you want to just delete this?

**Comment by mkhatib** [comment]
Nice work!

File: web-client/app/views/main.html (hunk 2 above)
Comment (RashaHussein) [inline web-client/app/views/main.html]: I'll delete them on my next pull request.

**Pull request merged**